### DISCUSSION TITLE
Easy Score and multiple bars on the same horizontal line

### DISCUSSION THREAD

**Discussion by Ciroxxxx** [0xfe/vexflow#1361] [Q&A] [answered]
Hi everybody,
Is it possible to have multiple bars on the same horizontal line by using Easy Score? 
Or I am forced to use the low level API?
Many thanks

**Comment by 0xfe** [comment, marked as answer]
You can use the `System` class to manage your bars. See [bach_tests.ts](https://github.com/0xfe/vexflow/blob/master/tests/bach_tests.ts) for an example.

**Reply by Ciroxxxx** [reply]
I'll give it a try, many thanks
